web/styles/zulip.css

```diff
@@ -804,6 +804,7 @@ strong {
             padding: 3px 10px;
             gap: 5px;
             font-weight: normal;
+            max-width: 292px;
```

Comment:
Where does this number come from? We're trying to not add magic numbers in favor of using flex and CSS grid.

Reply:
We discussed that [here](https://github.com/zulip/zulip/pull/30640#discussion_r1660075651). It is needed to set a `max-width` for the typeahead for `text-overflow: ellipsis` to work.

Reply:
OK but the [conclusion](https://github.com/zulip/zulip/pull/30640#discussion_r1660514421) there was to add a comment explaining this number, which I don't see?

Reply:
Added a comment.